lib/linguist/languages.yml

```diff
@@ -2176,6 +2176,16 @@ Git Attributes:
   codemirror_mode: shell
   codemirror_mime_type: text/x-sh
   language_id: 956324166
+Git Blame Ignore:
```

Comment:
I'd name this `Git Revision List` instead. Both `git blame --ignore-revs-file`[^1] and the `fsck.skipList` config setting use the same file-format, despite being two fundamentally different operations. From [`Documentation/config/fsck.txt`](https://github.com/git/git/blob/bc42e1e767fa86261fb5fdc5a495ad3ee14a9e0b/Documentation/config/fsck.txt):


> <dl><dt><b>fsck.skipList::</b></dt><dd>The path to a list of object names (i.e. one unabbreviated SHA-1 per line) that are known to be broken in a non-fatal way and should be ignored. On versions of Git 2.20 and later comments ('#'), empty lines, and any leading and trailing whitespace is ignored. Everything but a SHA-1 per line will error out on older versions.</dd></dl>

[^1]: https://git-scm.com/docs/git-blame#Documentation/git-blame.txt---ignore-revs-fileltfilegt <q>Ignore revisions listed in <var>file</var>, which must be in the same format as an <code>fsck.skipList</code></q>

Reply:
Does fsck-skiplist have a canonical filename?

Reply:
Nope, none.